test/plugin_functional/test_suites/core_plugins/rendering.ts

```diff
@@ -220,6 +220,8 @@ export default function ({ getService }: PluginFunctionalProviderContext) {
         'xpack.trigger_actions_ui.enableGeoTrackingThresholdAlert (boolean)',
         'xpack.upgrade_assistant.readonly (boolean)',
         'xpack.upgrade_assistant.ui.enabled (boolean)',
+        'xpack.observability.unsafe.alertDetails.enabled (boolean)',
```

Comment:
We might want to add an entry in the [readme](https://github.com/kdelemme/kibana/blob/main/x-pack/plugins/observability/README.md#L22) file about this feature flag.
Also do we need to have the flag in the [kibana-docker](https://github.com/kdelemme/kibana/blob/main/src/dev/build/tasks/os_packages/docker_generator/resources/base/bin/kibana-docker#L297) as well (not sure)?

Reply:
@kdelemme, thanks for your comment! Indeed, I will add them now